samples/base/Session.cpp

```diff
@@ -435,10 +446,10 @@ namespace darwin {
         return false;
     }
 
-    std::string Session::Evt_idToString(){
+    std::string Session::Evt_idToString() {
         char str[37] = {};
         sprintf(str,
```

Comment:
```suggestion
        snprintf(str, 36, 
```

Reply:
We need to test this before releasing it.

I will open an issue for the 1.0.1 milestone that concern the Core hardening.